packages/terra-application-links/src/tabs/ApplicationTabs.scss

```diff
@@ -149,33 +166,44 @@
     width: 100%;
     word-break: break-word;
     word-wrap: break-word;
-    
-    &:focus,
-    &.is-focused,
-    &:hover,
-    &.is-hovered,
-    &:visited,
-    &.is-visited {
-      border-radius: 0;
-      color: var(--terra-application-tabs-collapsed-color, #1c1f21);
-    }
+    z-index: 0;
 
     &:hover,
     &.is-hovered {
-      background: var(--terra-application-tabs-collapsed-hover-background, linear-gradient(-90deg, #fff, #f1f1f2));
+      background-color: var(--terra-application-tabs-collapsed-hover-background-color, #f5fafe);
     }
 
     &[aria-current='true'] {
-      background-image: var(--terra-application-tabs-collapsed-selected-background-image, linear-gradient(-270deg, #fff, #e8e9ea));
```

Comment:
webdriver test need to be updated to remove this themeable variable and [this screenshot](https://github.com/cerner/terra-framework/blob/3594f010039425034be854c5f15b6b6760e75479/packages/terra-application-links/tests/wdio/__snapshots__/reference/chrome/Displays_collapsed_application_tabs%5B--terra-application-tabs-collapsed-selected-background-image%5D.1000x768.png) needs removed: